bundles/admin/admin-layereditor/view/LayerWizard/LayerURLForm.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { UrlInput } from '../../components/UrlInput';
+import { Button } from '../../components/Button';
+
+const versionsAvailable = {
+    'WFS': ['3.0']
+};
+
+export class LayerURLForm extends React.Component {
+    constructor (props) {
+        super(props);
+        this.mutator = props.service;
+        this.state = {
+            url: props.layer.url
+        };
```

Comment:
Maximum one component should own the url state. I propose that this component would own the url state and that we make `UrlInput` a controlled component.
https://reactjs.org/docs/lifting-state-up.html